string iio_path # A valid IIO path to a device
---
bool success
string message
string[] data
